declarations/Twitter.json

```diff
@@ -5,10 +5,11 @@
       "fetch": "https://twitter.com/tos",
       "remove": [
         ".p05-chapter-navigation",
-        "[style='display: none;']"
+        "[style='display: none;']",
+        ".u02"
       ],
       "select": [
-        ".p22-chapters-container"
+        ".twtr-container"
```

Comment:
As the ToS page seems to share the same template as the Privacy Policy Twitter page, i suggest to use the same selector `main[id=twtr-main]` (the remove selectors should be updated accordingly)